Makefile

```diff
@@ -1,15 +1,33 @@
-all:
-	cd srcs && docker-compose up -d
 
-logs:
-	cd srcs && docker-compose logs
+NAME=Transcendence
 
-clean:
-	cd srcs && docker-compose down --remove-orphans
+$(NAME): build
```

Comment:
if two rules does exactly the same (here you say that all should call $(NAME) and nothing else) then all == $(NAME) therefore, that rant you just made is useless.
But i'll add it anyway.

Reply:
I did not intended it as a rent, i apologize,
having an extra rule in c/c++ usually is used to check if your executable is build or not.
Here since there is none, what might be easier, is to create a file upon building and checking if it exists.
Additionally this file could  be a log file to have an extra use.